djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
+
+    if version == 'latest':
```

Comment:
This is not doing what it sounds like it's doing. `latest` is a constantly-changing value; if you wanted to implement it, you'd have to actively *not* specify the version (I believe the stripe lib avoids it). I'd say strike it.